spec/spec_helper.rb

```diff
@@ -26,7 +26,8 @@
 def successful_body(app, options = {})
   retry_limit = options[:retry_limit] || 100
   path = options[:path] ? "/#{options[:path]}" : ''
-  Excon.get("http://#{app.name}.herokuapp.com#{path}",
+  web_url = app.send(:platform_api).app.info(app.name).fetch("web_url")
```

Comment:
alternatively, couldn't this be integrated into hatchet after the app is created [here](https://github.com/heroku/hatchet/blob/main/lib/hatchet/app.rb#L327) and then exposed as a `web_url` field?